Parser/asdl_c.py

```diff
@@ -887,16 +887,18 @@ def visitModule(self, mod):
     if (PyObject_GetOptionalAttr((PyObject*)Py_TYPE(self), state->_fields, &fields) < 0) {
         goto cleanup;
     }
-    if (fields) {
-        numfields = PySequence_Size(fields);
-        if (numfields == -1) {
-            goto cleanup;
-        }
-        remaining_fields = PySet_New(fields);
+    if (fields == NULL) {
+        PyErr_Format(PyExc_TypeError,
+                     "%.400s has no fields",
+                     _PyType_Name(Py_TYPE(self)));
```

Comment:
```suggestion
        PyErr_Format(PyExc_TypeError,
                     "%T has no fields",
                     self);
```

I never remember whether we use `%T` or not in this file :D and whether we should for new code or not.